### ISSUE TITLE
Keybindings with some numbers not working

### ISSUE THREAD

**Issue by kleenkanteen** [infokiller/web-search-navigator#307]
Here is my keybindings:

![web search](https://user-images.githubusercontent.com/67878058/133027186-7f741024-dcf8-4c68-933b-19a57e719ba0.JPG)

So doing z 0 together does sort by last hour. But z 7, z 8, and z 9 do not work. I tried doing z+7 instead and so on for the rest but no banana. I have checked my other extensions and none use the same or any of the same numbers for shortcuts. Am bamboozled. Opened a new window, same issue.  

I tried changing them to just numbers, so z 0 to 0, z 7 to 7, and so on. Same issue, only 0 works but not 7 or 8 or  9.

**Comment by infokiller** [comment]
I can't reproduce this issue. Please follow https://github.com/infokiller/web-search-navigator#troubleshooting and report back

**Comment by kleenkanteen** [comment]
When troubleshooting I get this error in the second part:

```
~/Documents/web search test/web-search-navigator (master)
$ yarn install && yarn build
yarn install v1.22.11
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.2.13: The platform "win32" is incompatible with this module.
info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
error eslint-visitor-keys@3.0.0: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.17.0 || >=16.0.0". Got "15.14.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

~/Documents/web search test/web-search-navigator (master)
$ yarn run web-ext run --source-dir src --target chromium
yarn run v1.22.11
error Command "web-ext" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I tried doing this and got this error:

`$ yarn add web-ext
yarn add v1.22.11
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.2.13: The platform "win32" is incompatible with this module.
info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
error eslint-visitor-keys@3.0.0: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.17.0 || >=16.0.0". Got "15.14.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.`

Any idea why?

**Comment by infokiller** [comment]
I don't know but it seems like the nodejs version you have installed is incompatible. Maybe try installing one of the "Expected" versions from the error message.
Alternatively, you can just create a new browser profile (check Chrome docs if needed) and only add this extension.

**Comment by kleenkanteen** [comment]
Ok I downloaded the LTR version of node, 14.17.6. Also installed the visual studio build tools which I hadn't last time. maybe that was the issue. So now doing troublshooting works when I wrote `yarn run web-ext run --source-dir src --target chromium`. And the numbers worked as shortcuts. I went to my main browser and disabled vimium and now it works fine. It is odd though because in vimium I unmapped all the keys, then added the ones I want, and 1 2 3 4 are used but 5 6 7 8 9 0 are not, but the unused numbers are blocking this extension it seems. I will ask in the vimium github why it's happening and report back.
